### ISSUE TITLE
[Medium Priority] Built-in blocklist is lacking some rules and can't be customized.

### ISSUE THREAD

**Issue by gettysburg** [henrypp/simplewall#785]
OS: Server 2019 
Kernel: 1809 17763.1490

The built-in block-lists lack some rules for new subnets and IP addresses which Microsoft use to phone-home to.

For example, when opening and then closing Windows Update with all three blacklists (Spying & Telemetry, Update Servers, Application Servers) enabled, only some addresses are on the blacklists, some are missing and then get blocked by the catch-all filter:

![image](https://user-images.githubusercontent.com/2256003/94543031-12660780-0239-11eb-8607-bbfa586730ad.png)

If needed, I might compile a list with all subnets and IP addresses I stumble upon - but please notice that they might not be the same across different Windows versions.

**Comment by alext6565** [comment]
but still get blocked. lol

**Comment by gettysburg** [comment]
> 
> 
> but still get blocked. lol

Doesn't matter, some people might not have the "Block all outbound connections" filter enabled.

**Comment by gettysburg** [comment]
I'll close this for now, seeing that I totally didn't know that checking for updates inside simplewall also tends to update the "hard-coded" filters.

This ticket would otherwise stay open forever, given that OS's change with almost every update, and CDN's and other IP's are changed quite often.

Now that the bug resulting in updates not showing due to a wrong time constant has been fixed (#745), the updates from the WindowsSpyBlocker repository can actually be received and merged into the app's block lists.

Thank you @henrypp for your outstanding WFP frontend, and @crazy-max for taking the time to stay on top of Microsoft's spying game.
